server/sonar-server/src/main/java/org/sonar/ce/log/CeProcessLogging.java

```diff
@@ -28,7 +29,7 @@
 public class CeProcessLogging extends ServerProcessLogging {
 
   public CeProcessLogging() {
-    super("ce", "%X{ceTaskUuid}");
+    super(LogbackHelper.LogProcess.CE, "%X{ceTaskUuid}");
```

Comment:
conflicting change with my PR, not so hard to fix, but I'd rather let you handle it since I have 5 branches with that change :)